src/main/resources/swagger/api-docs.yaml

```diff
@@ -2548,7 +2548,9 @@ components:
       x-tokenName: id_token
       flows:
         implicit:
-          authorizationUrl: https://tdrb2ctest.b2clogin.com/tdrb2ctest.onmicrosoft.com/oauth2/v2.0/authorize?p=B2C_1A_SIGNUP_SIGNIN
+          # This points at the B2C dev environment. See https://broadworkbench.atlassian.net/browse/TOAZ-85
```

Comment:
Noting this ticket to improve the Swagger experience -- for now, just switching the B2C URL to the new dev instance.